about.html

```diff
@@ -0,0 +1,121 @@
+                   culture and be proud of it.
+               </p>
+           </div>
+           <div class="container text-center">
+           <span><strong>Our Mission:</strong> To promote Rwandan arts in order to be recognised on
+                                            international level.</span><br>
+           <span><strong>Our vision:</strong> To be the first country in Africa with recognisable
+                                             and influencing cultural heritage.</span>
+           </div>
+        </div>
+       </section>
+
+       <section>
+           <div class="container-fluid  pb-0 text-center" id="festival">
+           <h3 class="underline-small about-2">About show me <br> what you got festival tour</h3>
+           <div class="container  d-flex py-4">
+               <div class="row">
+                   <div class="col-sm-3 col-lg-6 "><img src="./images/Show-me-festival.jpg" alt="photo of show me what you got festival" class="festiv-img"></div>
+                   <div class=" showcase-text col-sm-3 col-lg-6  py-2">
+                       <p class=" main-paragraph m-0 d-lg-flex flex-column align-content-center">In order to support Rwandan artists, Umuhanzi group has launched <strong>show me what
+                           you got festival tour</strong> which will give those artists the opportunity to perform live in front of their fans, 
+                            and observe how people love their songs. <br>
+                            For the people who will be attending, they will enjoy the quality of Rwandan music, sing and dance 
+                            together with their favorite stars. These festivals will be taking place in various places across the country. <br>
+                            Let us support our artists by loving and supporting their arts.
+                       </p>
+                   </div>
+               </div>
+           </div>
+        </div>
+       </section>
+
+       <section>
+           <div class="container-fluid text-center" id="other-projects">
+           <h3 class="underline-small about-3">See our past events</h3>
+           <p class="project-paragraph">Umuhanzi group has been organising various events since 2018, in order to support Rwandan artists.
+               See our past events.
+           </p>
+          <div class="container ">
+            <div class="row d-flex justify-content-center">
+               <div class="col-sm-3 p-4 m-3 project-img project-img-1"><h3>Iwacu muzika festival</h3><span>Season 1 in 2018</span><div class="overlay"></div></div>
+               <div class="col-sm-3 p-4 m-3 project-img project-img-2"><h3>Kigali fiesta</h3><span>summer festival</span><div class="overlay"></div></div>
+               <div class="col-sm-3 p-4 m-3 project-img project-img-3"><h3>New Year festival</h3><span>yearly festival</span><div class="overlay"></div></div>
```

Comment:
- [ ] As per the design, only 2 images with text content are given. You provided 3 in your design. Please remove one.